services/venice-controller/src/main/java/com/linkedin/venice/controller/kafka/consumer/AdminConsumptionTask.java

```diff
@@ -575,16 +579,22 @@ private void executeMessagesAndCollectResults() throws InterruptedException {
 
               if (lastSucceededId == newLastSucceededId && perStorePendingMessagesCount > 0) {
                 // only mark the store problematic if no progress is made and there are still message(s) in the queue.
-                errorInfo.exception = new VeniceException(
-                    "Could not finish processing admin message for store " + storeName + " in time");
-                errorInfo.offset = storeAdminOperationsMapWithOffset.get(storeName).peek().getOffset();
-                problematicStores.put(storeName, errorInfo);
-                LOGGER.warn(errorInfo.exception.getMessage());
+                AdminOperationWrapper nextOperation = storeAdminOperationsMapWithOffset.get(storeName).peek();
+                if (nextOperation != null) {
```

Comment:
Similarly here, might be worth still reporting the issue just without the offset if we caught a `CancellationException` but the admin operation is somehow null.

Reply:
In https://github.com/linkedin/venice/pull/1281/commits/2cc911156125577d647b6a1921095505b300f473 I have made it so that the logging always happens.